tests/cromwellapi/test-validate.py

```diff
@@ -1,25 +1,35 @@
+from pathlib import Path
+
 import pytest
-from utils import path_options, path_wdl
+
+root = Path(__file__).parents[2].resolve()
+pattern = "**/*.wdl"
+wdl_paths = list(root.glob(pattern))
```

Comment:
@sckott is this the best way to get the wdl paths?

Reply:
THIs is how I do it in conftest.py for the `submit_wdls` fixture https://github.com/FredHutch/wdl-unit-tests/blob/main/tests/cromwellapi/conftest.py#L61-L63 - Maybe there's a better way?

Reply:
Just wanted to double check, thanks. In that case LGTM.